clusterloader2/pkg/measurement/common/network/controller.go

```diff
@@ -0,0 +1,438 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package network
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"math"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	clientset "k8s.io/client-go/kubernetes"
+	measurementutil "k8s.io/perf-tests/clusterloader2/pkg/measurement/util"
+
+	"k8s.io/klog"
+)
+
+var workerPodList map[string][]workerPodData
+
+var firstClientPodTime int64
+
+const initialDelayForTCExec = 5
+
+var metricVal map[string]MetricResponse
+var uniqPodPairList []uniquePodPair
+var metricRespPendingList []uniquePodPair
+var k8sClient clientset.Interface
+
+var networkPerfRespForDisp NetworkPerfResp
+
+const WorkerListenPort = "5003"
+
+var podPairCh chan uniquePodPair
+
+func (npm *networkPerfMetricsMeasurement) startCtrl() {
```

Comment:
should this be with the rest of methods and the object definition in `clusterloader2/pkg/measurement/common/network/network_performance_measurement.go`?